### ISSUE TITLE
Support multi-byte register addresses for _data() functions

### ISSUE THREAD

**Issue by gnbl** [EmbedME/pyI2C_MP_USB#1]
The register address in `_data()` functions appears to be limited to a single byte, which is fine for the 2kbit (256 byte) EEPROM example (https://github.com/EmbedME/pyI2C_MP_USB/blob/master/examples/eeprom_test.py).

As a workaround, I've modified the respective USB controlwrite() function calls to unpack the `*register`, which allows passing multiple address bytes as a list (iterable) (e.g. in https://github.com/EmbedME/pyI2C_MP_USB/blob/master/i2c_mp_usb/i2c_mp_usb.py#L183).
Of course this breaks the current interface, but it should by possible to support both. Alternatively, additional methods analogous to the "word" transfer functions could be implemented for two-byte register addresses.

Or am I missing something? :-)

Regardless, thanks for sharing this, I'm using it with [Till Harbaums firmware](https://github.com/harbaum/I2C-Tiny-USB/tree/master/digispark) on a Digispark (ATtiny85 USB board).

**Comment by gnbl** [comment]
I suppose it could be done with write_i2c_block_raw() followed by read_i2c_block_raw()

**Comment by AntonGue** [comment]
Thank you for this issue, I had the same problem with MLX90640 sensor, and was looking for a solution that doesn't require an override. I'll try the read/write_i2c_block_raw methods.

Edit for anyone who's looking for help later:
I managed to read from the mlx90640. I extended the class and create my own method, which is basically the 'read_word_data', except that I removed the [ ] brackets around 'register' in the 'controlWrite' call and added the 'CMD_I2C_IO_BEGIN' constant to the 'controlRead' call.
